roles/setup_server/meta/main.yml

```diff
@@ -0,0 +1,52 @@
+galaxy_info:
```

Comment:
these `meta/main.yml` files need to be filled out with the minimal required metadata (we don't need most of this information since it's inferred from the collection metadata)

an example here: https://github.com/ansible-collections/community.sops/blob/561f443ddb85fa1dae9351d873b673ad03868320/roles/install/meta/main.yml

(note the `standalone: false`, which means this role is a part of a collection instead of being a "standalone" role)